python/mxnet/symbol/random.py

```diff
@@ -247,3 +247,34 @@ def multinomial(data, shape=_Null, get_prob=True, **kwargs):
         reward as head gradient w.r.t. this array to estimate gradient.
     """
     return _internal._sample_multinomial(data, shape, get_prob, **kwargs)
+
+
+def shuffle(data, **kwargs):
```

Comment:
Python interface seems unnecessary. You can register the operator with name `_random_shuffle` in C++.